### PR TITLE
Transform static methods into classes

### PR DESCRIPTION
**Pull request by victornoel**
See #440.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#442](https://github.com/yegor256/jpeek/issues/442) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/55a9e1d8-0a24-4190-89fe-5434c7764c82); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/org/jpeek/graph/XmlGraph.java [hunk 1]
```diff
@@ -104,28 +105,61 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
...
+     * @todo #440:30min This class XMLCallText and the following one
+     *  XMLMethodArgsText should be made public and tests should be added
+     *  to validate both their behaviours.
      */
-    private static String call(final XML call) throws IOException {
-        return new Joined(
-            "", call.xpath("name/text()").get(0),
-            ".", XmlGraph.args(call)
-        ).asString();
+    private static final class XmlCallText implements Text {
```
Comment (Vatavuk) [inline src/main/java/org/jpeek/graph/XmlGraph.java:114]: @victornoel I believe `Text` suffix is redundant. `XmlCall` sound cleaner.

File: src/main/java/org/jpeek/graph/XmlGraph.java [hunk 2]
```diff
@@ -104,28 +105,61 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
...
      * Serialize method arguments to a string.
-     * @param method Method as XML
-     * @return A string representation of the method args
-     * @throws IOException If fails
+     *
+     * @since 1.0
      */
-    private static String args(final XML method) throws IOException {
-        return new Joined(":", method.xpath("args/arg/@type")).asString();
+    private static final class XmlMethodArgsText implements Text {
```
Comment (Vatavuk) [inline src/main/java/org/jpeek/graph/XmlGraph.java:144]: @victornoel I believe `Text` suffix is redundant. `XmlMethodArgs` sound cleaner.

**Comment by victornoel** [comment]
@Vatavuk thank you, it is done

**Comment by Vatavuk** [comment]
@paulodamaso good to merge

**Comment by paulodamaso** [comment]
@victornoel Please extract them from XmlGraph and make them not static.

**Comment by victornoel** [comment]
@paulodamaso it is done, thank you

**Comment by paulodamaso** [comment]
@victornoel Thanks!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by victornoel** [comment]
@paulodamaso I just fixed the todos

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make you will find 3 issues during next CR, thx

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**